### PR TITLE
server-client example tutorial

### PR DESCRIPTION
**Pull request by GabrielaReyna**

### CODE REVIEW

**Comment by GabrielaReyna** [comment]
The lint warning is an example address in a code line. Is not a real link, should I add it to the list anyway?

**Comment by nanaian** [comment]
> The lint warning is an example address in a code line. Is not a real link, should I add it to the list anyway?

Yes, we should ignore all loopback addresses. I think this isn't picked up because the port number is different

File: src/content/docs/cheerpj3/13-examples/04-serverclient.mdx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+http-server -p 8080
+```
+
+4. Now open a browser tab and visit the address you are serving. It should be something similar to `http://127.0.0.1:8080/server.html`.
+5. Above the application display you will see the assigned IP address for the Server app. Please reserve it as you will need it later to connect the client with the server. You can also consult the IP address by visiting the [Tailscale Dashboard](https://login.tailscale.com/admin/machines).
+6. You will see the CheerpJ loading animation for a brief moment and your application should show.
+
+### 3.2 The Client application
+
+1. Open the `client.html`file and paste the following code:
```
Comment (nanaian) [inline src/content/docs/cheerpj3/13-examples/04-serverclient.mdx:119]:
```suggestion
1. Open the `client.html` file and paste the following code:
```

File: src/content/docs/cheerpj3/13-examples/04-serverclient.mdx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+- [`cheerpjCreateDisplay`] creates a graphical environment to contain all Java windows.
+- [`cheerpjRunJar`] executes your application!
+- [`tailscaleIpCb`] is used to retrieve the IP address once the application is connected to the Tailscale network. We are using it to clearly display the IP address on the document.
+- `/app/` is a [virtual filesystem] mount point that references the root of the web server this page is loaded from.
+
+2. Copy the first auth key you generated in the previous step and paste it as a string replacing _"PasteYourKeyHere"_.
+3. Now you are ready to serve this page and run the Server application in the browser. Simply run a http-server **inside** the `Server` directory. For example:
+
+```bash
+cd /ServerClientTutorial/Server/
```
Comment (nanaian) [inline src/content/docs/cheerpj3/13-examples/04-serverclient.mdx:109]: Not sure about the absolute path here. Perhaps just remove the prefixing `/`?

File: src/content/docs/cheerpj3/13-examples/04-serverclient.mdx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+**What is happening here?**
+
+- [`cheerpjRunJar`] The second parameter corresponds to the _args_ object which is an argument for the Java Main method. we are passing the API address of the Server application so the Client can communicate with it.
+
+2. Copy the second auth key you generated previously and paste it as a string replacing _"PasteYourKeyHere"_.
+3. Now you are ready to serve this page and run the Client application in the browser. Simply run a HTTP server **inside** the `Client` directory with a different port than the one used for the Server app. For example:
+
+```bash
+cd /ServerClientTutorial/Client/
```
Comment (nanaian) [inline src/content/docs/cheerpj3/13-examples/04-serverclient.mdx:158]: "

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**